### ISSUE TITLE
can't get app to launch on my machines

### ISSUE THREAD

**Issue by tokyomegaplex** [alievk/avatarify-desktop#32]
Hello! Running windows 10 with latest nvidia drivers on two machines. 

One has two 3090s and the other has two a5000s. 

I don't get anything to show up at all when i launch the app, but it displays as running in background on ask manager. Any idea what could be wrong?

Thanks!

**Comment by HiramIII** [comment]
did you ever figure out a fix?

**Comment by tokyomegaplex** [comment]
nope, no idea about this :/

**Comment by caesarkkim** [comment]
> 你有没有想过解决办法？

helppppppp, me to

**Comment by JohanAR** [comment]
The app does not work on 30XX cards due to some package incompatibility, see #9

**Comment by slyfox63** [comment]
I got the same problem when I use a laptop with a 3060 and the latest nvidia drives

**Comment by AlexMadgwick** [comment]
yup same issue here. camera light is on background task is running but no app 3080 card.

**Comment by AlexMadgwick** [comment]
also tried turning off optimums and ruining as admin, no dice.
